listenbrainz_spark/recommendations/train_models.py

```diff
@@ -1,3 +1,23 @@
+# This script is responsible to train models and save the best model to HDFS. The general
```

Comment:
Again this just seems like a description of the functions, it would be better if the docstring explained in plain english what the script was doing instead of just explaining each function